portal/templates/login/_macros/contact-us.njk

```diff
@@ -0,0 +1,15 @@
+{% macro render(params) %}
+  {# This is very similar to _partials/contact-us.njk #}
```

Comment:
Will remove this comment, just wanted to flag whether I should be using this partial rather than building this new macro. The design is slightly different: missing the line  `...If you have a unique reference number...`, and using `Customer service team` rather than `Telephone`.

I also noticed the existing partial uses `<h2 class="govuk-heading-s">Email</h2>` for one heading and `<h3 class="govuk-heading-s">Telephone</h3>` for the other

Reply:
I’ve discussed with Adam : The contact us section of this page is going to match existing contact us sections, and include:

<p class="govuk-body">If you have a unique reference number, have it with you when you call.</p>


above Telephone section (for both the 3rd email resend page and the account blocked page)